docs/development/firmware/building.mdx

```diff
@@ -62,7 +62,7 @@ The build system is modular. Adding a new board variant for an already supported
 
 1. Perform all of the steps [building with custom hardware](#build-with-custom-hardware) until your hardware runs fine.
 2. [Send a proposal](https://github.com/meshtastic/firmware/issues/new?assignees=&labels=enhancement%2Ctriage&template=New+Board.yml&title=%5BBoard%5D%3A+) to add a new board.
-3. If approved, go to (https://github.com/meshtastic/protobufs) and send a Pull Request for the `mesh.proto` file, adding your board to the `HardwareModel` Enum.
+3. If approved, go to <a href="https://github.com/meshtastic/protobufs">https://github.com/meshtastic/protobufs</a> and send a Pull Request for the `mesh.proto` file, adding your board to the `HardwareModel` Enum.
```

Comment:
Any reason to not use markdown here?
[https://github.com/meshtastic/protobufs](https://github.com/meshtastic/protobufs)
<img width="552" alt="Screenshot 2024-08-24 at 7 51 05 AM" src="https://github.com/user-attachments/assets/7a3383df-6e79-423f-952a-89f77ca5bd4f">

Reply:
agreed - I'm getting carried away.